src/base/icons/icons.json

```diff
@@ -1,6 +1,6 @@
 {
-  "activity": "fas fa-shopping-basket",
-  "activity_fw": "fas fa-fw fa-shopping-basket",
+  "activity": "fas fa-asterisk",
+  "activity_fw": "fas fa-fw fa-asterisk",
```

Comment:
Check the other override group themes (and locales...)

Reply:
Could remove both the icon overrides (and switched to also use circle icon for places, was shopping basket before for foodsaving groups).

Also, removed most of the general purpose locale override, but kept the place status overrides...